### PR TITLE
Adds unit tests for add site component

### PR DESCRIPTION
**Pull request by el-mapache**
* Adds unit tests for AddSite component

### CODE REVIEW

File: test/client/app/components/addSite/addSite.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    wrapper.instance().onSubmitTemplate(templateRepo, siteName);
+
+    expect(cloneRepo.calledWith({repo: templateRepo}, templates[siteName])).to.be.true;
+    expect(cloneRepo.called).to.be.true;
+  });
+
+  it('delivers the correct props to its children', () => {
+    const bannerProps = wrapper.find(AlertBanner).props();
+    const templateListProps = wrapper.find(TemplateSiteList).props();
+    const linkButtonProps = wrapper.find(LinkButton).props();
```
Comment (brandocalrissian) [inline test/client/app/components/addSite/addSite.test.js]: Is the intent to test if the right props are given to the link button? These props aren't validated in the test.

Comment (el-mapache) [inline test/client/app/components/addSite/addSite.test.js]: Oops, yes, I left that in by mistake. I was going to test that those props were being correctly passed, but they are hardcoded, not computed. That made me think it was an unnecessary test. Does that seem reasonable?

Comment (brandocalrissian) [inline test/client/app/components/addSite/addSite.test.js]: Yeah, testing configuration is not a very interesting test as you're simply copying the same strings from one place to another. I vote for taking out the line.

Comment (el-mapache) [inline test/client/app/components/addSite/addSite.test.js]: word, updated!

**Comment by brandocalrissian** [comment]
Looks good! Merging...

**Pull request merged**